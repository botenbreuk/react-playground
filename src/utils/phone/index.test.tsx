import { shallow } from 'enzyme';
import { PhoneNumber } from './index';

describe('Util: string', () => {
  it('formatPhone should create tel link', () => {
    const phoneNumberNull = shallow(<PhoneNumber />);
    expect(phoneNumberNull).toMatchSnapshot();

    const phoneNumberEmpty = shallow(<PhoneNumber phone="" />);
    expect(phoneNumberEmpty).toMatchSnapshot();

    const phoneNumber = shallow(<PhoneNumber phone="31612345678" />);
    expect(phoneNumber).toMatchSnapshot();
  });
});
