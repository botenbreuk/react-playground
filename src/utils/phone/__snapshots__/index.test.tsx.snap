// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Util: string formatPhone should create tel link 1`] = `ShallowWrapper {}`;

exports[`Util: string formatPhone should create tel link 2`] = `ShallowWrapper {}`;

exports[`Util: string formatPhone should create tel link 3`] = `ShallowWrapper {}`;
